Vagrantfile

```diff
@@ -303,7 +303,7 @@ Vagrant.configure('2') do |config|
   end
 
   config.vm.define :f_win2012r2 do |cfg|
-    cfg.vm.box = 'opentable/win-2012r2-standard-amd64-nocm'
```

Comment:
I tried running vagrant against this and got a 403 error.  It seems like this box no longer exists.